include/linux/seg6_local.h

```diff
@@ -16,17 +16,17 @@
 
 #include <linux/seg6.h>
 
-enum {
-	SEG6_LOCAL_UNSPEC,
-	SEG6_LOCAL_ACTION,
-	SEG6_LOCAL_SRH,
-	SEG6_LOCAL_TABLE,
-	SEG6_LOCAL_NH4,
-	SEG6_LOCAL_NH6,
-	SEG6_LOCAL_IIF,
-	SEG6_LOCAL_OIF,
-	SEG6_LOCAL_BPF,
-	__SEG6_LOCAL_MAX,
+enum { SEG6_LOCAL_UNSPEC,
```

Comment:
this is really odd formatting.  Why the change?

Reply:
Maybe I just followed frrbot, I'll fixing

Reply:
Here is the error from frrbot: https://github.com/FRRouting/frr/pull/9649/checks?check_run_id=4287347514